### PR TITLE
refactor(Hint): Convert hint examples to typescript

### LINKED ISSUE #7482: Convert Hint examples to typescript

### PR DESCRIPTION
**Pull request by bartoval**
**What**: Closes #7482

### CODE REVIEW

File: packages/react-core/src/components/Hint/examples/HintBasicWithoutTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  HintBody,
+  HintFooter,
+  Button,
+  Dropdown,
+  DropdownItem,
+  DropdownSeparator,
+  KebabToggle
+} from '@patternfly/react-core';
+
+export const BasicHint: React.FunctionComponent = () => {
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Hint/examples/HintBasicWithoutTitle.tsx:13]: Similar nitpick to the previous comment re: the exported component name matching the file name

**Review by thatblindgeye — COMMENTED** [review]
Only had a couple of nitpicks below, but otherwise it's looking good!

**Review by thatblindgeye — APPROVED** [review]

**Review by wise-king-sullyman — APPROVED** [review]
Looks great! 🚀

**Review by tlabaj — APPROVED** [review]
LGTM

**Pull request merged**